tests/integration/helpers/factories/db/invenio_records.py

```diff
@@ -55,7 +55,7 @@ class TestRecordMetadata(TestBaseModel):
     }
 
     @classmethod
-    def create_from_kwargs(cls, index=True, has_pid=True, **kwargs):
+    def create_from_kwargs(cls, index_name='', **kwargs):
```

Comment:
Mh, probably you've already discussed this, but why isn't the default `records-hep` ?

Reply:
The default is None for the case where we don't want to index the record in ES.

Reply:
Ah, yes, makes sense. But then why isn't the default `None`, instead of empty string? : )

Reply:
Empty string since it will contain string arguments (as suggested by Paolo).